src/docs/src/ddocs/nouveau.rst

```diff
@@ -584,13 +584,16 @@ Nouveau Search also supports faceted searching, enabling discovery of aggregate
 information about matches quickly and easily. You can match all documents by using the
 special ``?q=*:*`` query syntax, and use the returned facets to refine your query.
 
-*Example of search query:*
+*Example of faceted index:*
 
 .. code-block:: javascript
 
     function(doc) {
-        index("string", "type", doc.type);
-        index("double", "price", doc.price);
+        // not faceted
+        index("string", "email", doc.email, { facet: false })
```

Comment:
facet:true isn't recognised, all strings and doubles are indexed for faceting now.

Reply:
Doh. Will remove that commit :)